questdb-rs/src/ingress/mod.rs

```diff
@@ -1337,6 +1348,28 @@ fn add_webpki_roots(root_store: &mut rustls::RootCertStore) {
     }));
 }
 
+#[cfg(feature = "tls-native-certs")]
+fn add_os_roots(root_store: &mut RootCertStore) -> Result<()> {
+    let os_certs = rustls_native_certs::load_native_certs().map_err(|io_err| {
+        error::fmt!(
+            TlsError,
+            "Could not load OS native TLS certificates: {}",
+            io_err
+        )
+    })?;
+
+    let os_certs: Vec<Vec<u8>> = os_certs.into_iter().map(|cert| cert.0).collect();
+    let (valid_count, invalid_count) = root_store.add_parsable_certificates(&os_certs[..]);
+    if valid_count == 0 && invalid_count > 0 {
+        return Err(error::fmt!(
+            TlsError,
+            "zero valid certificates found in native root store ({} found but were invalid)",
```

Comment:
nit: "zero valid certificates" sounds strange to me.
I would personally use "no valid certificate found .." but up to you, I understand it's subjective.

Reply:
Done.